test/cmd/images_tests.sh

```diff
@@ -66,9 +91,9 @@ os::cmd::expect_success "oc annotate --context='${cluster_admin_context}' --over
 os::cmd::expect_failure_and_text "oc run vulnerable --image=${repository}:new --restart=Never" 'spec.containers\[0\].image: Forbidden: this image is prohibited by policy'
 
 # test image stream tag operations
-os::cmd::expect_success_and_text 'oc get istag/wildfly:latest -o jsonpath={.generation}' '2'
```

Comment:
Are these test still valid with your changes, given you're having to change what they're testing for? Also, are they doing something worthwhile? Why is `5` an important generation? It seems a little weird that we're testing `oc get ist` without *doing* something to them first.